### PR TITLE
News migration

### PR DESCRIPTION
**Pull request by sbesson**
See https://trello.com/c/aTdBNblP/18-news-migration

Migrates some of the October 2016 releases to the new website

### CODE REVIEW

File: _posts/2016-10-04-omero-5.2.6.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+-  fixed closure of session in Java when using Ice 3.5
+-  fixed memory leak where services were not correctly closed
+-  added a DB upgrade patch to fix errors only affecting DBs that have been upgraded from OMERO 4.4[1]
+-  fixed a MATLAB regression introduced in 5.2.2, casting error.
+-  fixed error in logs on getProjectedThumbnail
+ 
+Support for OMERO.web deployment using Apache has also been deprecated and is likely to be removed during the 5.3.x line.
+
+The software is available at
+[http://downloads.openmicroscopy.org/omero/5.2.6](http://downloads.openmicroscopy.org/omero/5.2.6)
```
Comment (kennethgillen) [inline _posts/2016-10-04-omero-5.2.6.md]: Should that be `[archived downloads](...)`

File: _posts/2016-10-04-omero-5.2.6.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Release of OMERO 5.2.6
+intro-blurb: The OME team is pleased to announce the release of OMERO 5.2.6
+---
+
```
Comment (kennethgillen) [inline _posts/2016-10-04-omero-5.2.6.md]: The example at https://github.com/openmicroscopy/www.openmicroscopy.org/pull/53/files
has no leading whitespace line.

File: _posts/2016-10-05-bio-formats-5.2.3.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Release of Bio-Formats 5.2.3
+intro-blurb: The OME team is pleased to announce the release of Bio-Formats 5.2.3
+---
+
```
Comment (kennethgillen) [inline _posts/2016-10-05-bio-formats-5.2.3.md]: as https://github.com/openmicroscopy/www.openmicroscopy.org/pull/64/files#r125048783

File: _posts/2016-10-05-bio-formats-5.2.3.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+*  Documentation
+    -  documented versioning policy
+    -  clarified supported versions for Micro-Manager and Olympus ScanR files
+*  OME-XML changes
+    -  bumped schema version number to 2 (schema namespace left unchanged)
+    -  added acquisition modes BrightField, SweptFieldConfocal and SPIM
+    -  added parsing for Laser Scan Confocal and Swept Field Confocal
+
+The software is available to download now at 
+[http://downloads.openmicroscopy.org/bio-formats/5.5.1](http://downloads.openmicroscopy.org/bio-formats/5.5.1).
```
Comment (kennethgillen) [inline _posts/2016-10-05-bio-formats-5.2.3.md]: Should be [archived downloads] cf https://github.com/openmicroscopy/www.openmicroscopy.org/pull/53/files#diff-639b1eb8b9357fce496f08267c893ef8R36

File: _posts/2016-10-12-ome-files-cpp-0.2.2.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Release of OME Files C++ 0.2.2
+intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.2.2
+---
+
```
Comment (kennethgillen) [inline _posts/2016-10-12-ome-files-cpp-0.2.2.md]: Remove leading empty line

File: _posts/2016-10-12-ome-files-cpp-0.2.2.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+OME-XML and OME-TIFF features.  This will continue for the following release.
+
+Major changes in this release include:
+
+-   OME-TIFF reader and writer updates
+
+**Downloads**
+
+The software release is available at 
+[http://downloads.openmicroscopy.org/ome-files-cpp/0.2.2/](http://downloads.openmicroscopy.org/ome-files-cpp/0.2.2/).
```
Comment (kennethgillen) [inline _posts/2016-10-12-ome-files-cpp-0.2.2.md]: Should be [archived downloads] cf https://github.com/openmicroscopy/www.openmicroscopy.org/pull/53/files#diff-639b1eb8b9357fce496f08267c893ef8R36

File: _posts/2016-10-18-bio-formats-5.2.4.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+*  OME-TIFF
+    *  fixed regression when populating plane metadata
+*  CZI
+    *  populated series metadata with the scene/position information
+
+Full details can be found in the [Bio-Formats version history](http://www.openmicroscopy.org/site/support/bio-formats5.2/about/whats-new.html)
+
+The software is available at 
+[downloads.openmicroscopy.org/bio-formats/5.2.4/](http://downloads.openmicroscopy.org/bio-formats/5.2.4).
```
Comment (kennethgillen) [inline _posts/2016-10-18-bio-formats-5.2.4.md]: Should be [archived downloads] cf https://github.com/openmicroscopy/www.openmicroscopy.org/pull/53/files#diff-639b1eb8b9357fce496f08267c893ef8R36

**Comment by sbesson** [comment]
@kennethgillen 4635e26 should address your review comments

**Comment by sbesson** [comment]
Pushed

File: _posts/2016-10-05-bio-formats-5.2.3.md [hunk 8]
```diff
@@ -34,4 +34,5 @@ Other improvements:
 Any problems or comments, please use the
 [OME Forums or mailing lists]({{ site.baseurl }}/support/).
 
-[archived downloads](http://downloads.openmicroscopy.org/bio-formats/5.5.1)
\ No newline at end of file
+The software is available at:
+[archived downloads](http://downloads.openmicroscopy.org/bio-formats/5.5.1).
```
Comment (kennethgillen) [inline _posts/2016-10-05-bio-formats-5.2.3.md]: Should be 5.2.3 ?

**Pull request merged**